app/services/transit_service.rb

```diff
@@ -11,13 +11,8 @@ def get_url(url)
   end
 
   def get_trips(origin_lat, origin_lon, destination_lat, destination_lon)
-    get_url("/otp/plan?fromPlace=#{origin_lat}, #{origin_lon}&toPlace=#{destination_lat}, #{destination_lon}")
+    get_url("/v3/otp/plan?fromPlace=#{origin_lat},#{origin_lon}&toPlace=#{destination_lat},#{destination_lon}")
   end
-
-  # def get_trips(origin_lat, origin_lon, destination_lat, destination_lon)
```

Comment:
Are we using this at all? Or is it outdated? Can we get rid of this and clean it up?